src/components/displayDocuments/documentCard.tsx

```diff
@@ -8,7 +8,7 @@ const CardContainer = styled.div`
   box-shadow: 0px 0px 1px rgba(0, 0, 0, 0.04), 0px 0px 2px rgba(0, 0, 0, 0.06), 0px 4px 8px rgba(0, 0, 0, 0.04);
```

Comment:
Could you reduce the size of the thumbnail image ? =) (either reduce the size itself or lower quality a bit